internal/action/actions_posix.go

```diff
@@ -1,4 +1,4 @@
-// +build linux darwin dragonfly solaris openbsd netbsd freebsd
+// +build linux darwin dragonfly solaris openbsd netbsd freebsd zos
```

Comment:
What about...
https://github.com/zyedidia/micro/blob/1f51d0b9e2ae7722eda60c5a82de2ab1ea9b16a2/internal/action/terminal_supported.go#L1 & https://github.com/zyedidia/micro/blob/1f51d0b9e2ae7722eda60c5a82de2ab1ea9b16a2/internal/action/terminal_unsupported.go#L1
...?

Hm curious, `solaris` and `netbsd` aren't treated here?!

Reply:
@JoeKar, yeah I'm not sure exactly what is going on there.  The only use of RunTermEmulator I could find is in the fzf plugin (https://github.com/micro-editor/updated-plugins/blob/master/fzf/main.lua).  It checks for shell.TermEmuSupported and if run invokes shell.RunTermEmulator, otherwise invokes shell.RunInteractiveShell.  And if shell.RunInteractiveShell fails it invokes (local) fzfOutput.

The plugin works without any source change.  I think it's working with RunInteractiveShell.  But I really don't understand why. 

If I add zos to build for terminal_supported.go and !zos for terminal_unsupported.go I get "Unsupported operating system".  Which makes no sense to me, since that message is in terminal_unsupported.go, which it shouldn't be building.

Since it is working with no changes I figured I'd deal with it later.  But if you have any ideas, that would be great.

Reply:
It's relevant here...
https://github.com/zyedidia/micro/blob/1f51d0b9e2ae7722eda60c5a82de2ab1ea9b16a2/internal/action/termpane.go#L63
...and here too...
https://github.com/zyedidia/micro/blob/1f51d0b9e2ae7722eda60c5a82de2ab1ea9b16a2/internal/action/command.go#L965

Have you really build with `GOOS` being set to `zos`? According to...
https://go.dev/doc/install/source#environment
...it's isn't really documented, but in...
https://github.com/golang/go/blob/f43d9c40f382def04442898d7581402759bff36a/src/go/build/syslist.go#L32
...it's listed, so I assume it should work in the moment it's invoked with the correct `GOOS` environment variable for a fitting architecture.